hack/make/.integration-daemon-setup

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+bundle .ensure-emptyfs
+bundle .ensure-frozen-images
+bundle .ensure-httpserver
```

Comment:
This is only used in one place, right?  With the simpler `bundle ...` invocation, it seems like it's not worth splitting out IMO.